include/module.hpp

```diff
@@ -87,6 +94,13 @@ struct ModuleLoader {
 
     void InjectSymbols(sol::state& lua) {
         lua.set_function("log", &ModuleLoader::DefaultLogger, this);
+        for (const auto &item: m_external_symbols){
+            item->InjectSymbols(lua);
+        }
```

Comment:
To jest ok, ale na innym branchu jest inne rozwiązanie. Będzie trzeba zadecydować.

Reply:
tak jak wyzej